index.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-var dotProp = require('dot-prop');
+var dotPropGet = require('dot-prop').get;
```

Comment:
Keep `var dotProp = require('dot-prop');` and use `dotProp.get()` in the calls instead.